### PR TITLE
Ensure nuget.exe, symchk and msbuild paths are set correctly.

### PR DESCRIPTION
**Pull request by ravimeda**
Ensure nuget.exe, symchk and msbuild paths are set correctly for publishAll.ps1.

### CODE REVIEW

File: samples/UWPTest/publishAll.ps1 [hunk 1]
```diff
@@ -1,8 +1,61 @@
...
 foreach ($targetArch in $targetArchs)
 {
-    msbuild UWPTest.proj /p:TargetArch=$targetArch
-    msbuild UWPTest.proj /p:TargetArch=$targetArch /p:IsNetNative=true
-}
\ No newline at end of file
+    $msBuildPath = "$currentWorkingDirectory\Bin\MSBuild.exe"
+    $msBuildArgs = "UWPTest.proj /p:TargetArch=$targetArch /verbosity:diag /fileLoggerParameters:LogFile=.\BuildLog.txt;Append;Verbosity=diagnostic;Encoding=UTF-8"
+	
+    $p = Start-Process -FilePath $msBuildPath -ArgumentList $msBuildArgs -PassThru -Verbose -Wait -ErrorAction Continue
```
Comment (weshaggard) [inline samples/UWPTest/publishAll.ps1:48]: Why do you need to call Start-Process directly? Why not just `& $msbuildPath $msBuildArgs`

**Comment by weshaggard** [comment]
LGTM other then the one nit question.

File: samples/UWPTest/publishAll.ps1 (hunk 1 above)
Comment (ravimeda) [inline samples/UWPTest/publishAll.ps1:48]: `& $msbuildPath $msBuildArgs` that would work too.  Start-Process gives a better control.  For example, say the path to MSBuild is invalid then the process exit code will not be 1, and the displayed exception message will help investigate.

**Comment by ravimeda** [comment]
Thanks Wes.

**Pull request merged**